airbyte-integrations/connectors/source-veeqo/manifest.yaml

```diff
@@ -0,0 +1,4273 @@
+version: 5.12.0
+
+type: DeclarativeSource
+
+description: >-
+  Veeqo Airbyte connector for Veeqo enables seamless data integration between
+  Veeqo's inventory and order management platform and various data warehouses or
+  applications. It allows users to sync Veeqo data such as orders, products,
+  inventory levels, and more, making it easier to analyze and manage e-commerce
+  operations. This connector streamlines data workflows, ensuring up-to-date and
+  accurate information for better business insights.
+
+check:
+  type: CheckStream
+  stream_names:
+    - orders
+
+definitions:
+  streams:
+    tags:
+      type: DeclarativeStream
+      name: tags
+      retriever:
+        type: SimpleRetriever
+        paginator:
+          type: DefaultPaginator
+          page_size_option:
+            type: RequestOption
+            field_name: page_size
+            inject_into: request_parameter
+          page_token_option:
+            type: RequestOption
+            field_name: page
+            inject_into: request_parameter
+          pagination_strategy:
+            type: PageIncrement
+            page_size: 25
+            start_from_page: 1
+            inject_on_first_request: true
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: /tags
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path: []
+      primary_key:
+        - id
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/tags"
+    orders:
+      type: DeclarativeStream
+      name: orders
+      retriever:
+        type: SimpleRetriever
+        paginator:
+          type: DefaultPaginator
+          page_size_option:
+            type: RequestOption
```

Comment:
orders and other streams support incremental please implement this feature.
<img width="624" alt="image" src="https://github.com/user-attachments/assets/6ce63cc5-5cf1-4857-8626-688ee79ad6c5">